### DISCUSSION TITLE
Thank you!

### DISCUSSION THREAD

**Discussion by oneiros** [yippee-fun/phlex#386] [General]
I have been putting this of for too long now, but better late than never: Thank you @joeldrapper for phlex. It is absolutely amazing.

A couple of years back I dabbled a lot with https://luckyframework.org/ (a web framework for crystal, a ruby-like language) and I enjoyed its view layer a lot. I always wanted to have something like that for rails and even started my own attempt at writing such a library. But I gave up after rails' helper methods gave me too much headache :grimacing: 

Since then I settled on ViewComponent which is OK, but of course phlex, even in its current early state, is so much better.

So once again: Thank you and please keep up the good work!

**Comment by joeldrapper** [comment]
Thanks so much, that means a lot. 😊
